ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Node.hs

```diff
@@ -48,15 +45,15 @@ import           Ouroboros.Consensus.Shelley.Eras (EraCrypto)
 import           Ouroboros.Consensus.Shelley.Ledger
 import           Ouroboros.Consensus.Shelley.Ledger.Inspect ()
 import           Ouroboros.Consensus.Shelley.Ledger.NetworkProtocolVersion ()
+import           Ouroboros.Consensus.Shelley.Node.Praos
```

Comment:
INFO: previously, we couldn't import this module because it would introduce a cyclic dependency.